### DISCUSSION TITLE
正式なアプリ名を考える

### DISCUSSION THREAD

**Discussion by Hamumayo55** [jphacks/E_2202#35] [Ideas]
いい感じのアプリ名を募集しています！！
選ばれた人にはPOから豪華景品があるかも...?!

**Comment by soto0201** [comment]
- yukusi
- Xx_yukusi_xX

**Reply by soto0201** [reply]
ユクシーはポケモンで知識の神なのでこれでいきたい
https://zukan.pokemon.co.jp/detail/480

**Reply by kagari** [reply]
英語名 Yuxie ってかっこいいな

**Comment by kagari** [comment]
* GoodQuery

**Comment by soto0201** [comment]
- YouQuery

**Reply by soto0201** [reply]
こちらに決定いたしました！皆さんありがとうございました！

**Comment by kagari** [comment]
* errgle
* yougle

**Comment by soto0201** [comment]
アプリ名変わるならそのタイミングで僕以外の人でデプロイしたい
(デモの時にdyno切れて見れない可能性を消したいお気持ちがある)
